modules/ROOT/assets/attachments/sync-gateway-public.yaml

```diff
@@ -164,7 +164,23 @@ parameters:
   heartbeat:
     in: query
     name: heartbeat
-    description: Default is 0. Interval in milliseconds at which an empty line (CRLF) is written to the response. This helps prevent gateways from deciding the socket is idle and closing it. Only applicable to longpoll or continuous feeds. Overrides any timeout to keep the feed alive indefinitely. Setting to 0 results in no heartbeat.
+    description: |+
+      The `heartbeat` defines the interval (in milliseconds) at which an empty line (CRLF) is written to the response. It helps prevent gateways from deciding the socket is idle and closing it.
```

Comment:
instead of "gateways" how about "Sync Gateway"

Reply:
Makes sense ... applied in https://github.com/couchbase/docs-sync-gateway/pull/354/commits/1f206ed93a40d28926c122017ee633d3f65a1781